### DISCUSSION TITLE
question about the dashboard / GUI

### DISCUSSION THREAD

**Discussion by ovizii** [vriteio/vrite#41] [Q&A] [answered]
I've finally managed to get the self-hosted vrite up and running and am wondering about tis backend. 

Is there an option to somehow hide the "panel" marked red in this screenshot? 
That area either displays the "welcome to vrite section OR the settings once I enter the settings. It seems intrusive and unnecessary or am I missing something here?

![image](https://github.com/vriteio/vrite/assets/837902/20c516f0-f5ec-4d88-adf4-094f73001122)

![image](https://github.com/vriteio/vrite/assets/837902/9ab509e2-43ed-4db7-8c8c-8536f107d5f0)

I'm kinda stuck on this screen :-(

![image](https://github.com/vriteio/vrite/assets/837902/61e9a440-d4b8-4a98-bb01-1a5b0f9221b5)

**Comment by areknawo** [comment, marked as answer]
Once you select a content piece in the dashboard, this panel also serves you to configure related metadata, which IMO is quite important when managing content in the dashboard.

Also, it's where you control the Git integration (when activated) and installed extensions (on Vrite Cloud).

But yes - you can hide or resize it by hovering over, and then dragging the right edge (similarly to VS Code if you're familiar).
Some of it is described in the [early docs](https://docs.vrite.io/usage-guide/getting-started/) but I'm still working on a more detailed description.

Like I've said - you'll probably want to leave it open once you get a hold of things in the dashboard.
In the editor, using the "Zen mode" will likely be more preferred, as it centers your content and removes all distractions.

**Comment by areknawo** [comment]
Didn't really notice the second question 😅
Just click a "Content Piece Card" in the dashboard to select it.
I've already written a somewhat complete guide on it here (at least for the Kanban): https://docs.vrite.io/usage-guide/kanban-dashboard/
